### ISSUE TITLE
More than 3 lines with bold text renders in the preview with extra return

### ISSUE THREAD

**Issue by reginvaldr** [nea/MarkdownViewerPlusPlus#98]
## Issue description
Type:

**Data Centers are grouped into Availability Zones (backup & restore**
**Availability Zones are grouped into Regions (e.g. Ireland)**  
**High speed private links between Availability Zones**  
**Data Centers are undisclosed**  
**Edge locations are used for caching**

Renders as:
Data Centers are grouped into Availability Zones (backup & restore Availability Zones are grouped into Regions (e.g. Ireland)
High speed private links between Availability Zones

Data Centers are undisclosed

Edge locations are used for caching

### Markdown to reproduce
```
  Please post the markdown code producing the described issue, if any
```

### Screenshot
_If available_

### Environment
* Notepad++ _X.Y.Z (x86/x64)_
* MarkdownViewer++ _X.Y.Z_
* Windows _7/8/10 (x86/x64)_

**Comment by nea** [comment]
Hey @reginvaldr 

It is card to check this if not absolutely the same is parsed into this commen. Line breaks should only happen on double line breaks or double whitespaces at the end of a line.

Could you check that?


Thanks

**Comment by reginvaldr** [comment]
Yes, I understand. That is also what I expected. But after three lines
**text**[spatie][spatie] it starts rendering the 4th line with an empty
line (not just a line break)

So in Notepad type

**text**[space][space]
**text**[space][space]
**text**[space][space]
**text**[space][space]
**text**[space][space]

And you will see an empty line after 4th line


Op di 6 nov. 2018 om 16:15 schreef Savas Ziplies <notifications@github.com>

**Comment by nea** [comment]
Hi @reginvaldr 

Ok, it seems the HTMLRenderer does that. The exported HTML looks correct. As I do not maintain that library it will be a tough one to fix, but I see now.


Thanks
